src/lib/converter/nodes/variable-statement.ts

```diff
@@ -42,10 +41,14 @@ export class VariableStatementConverter extends ConverterNodeComponent<ts.Variab
      * @param node     The binding pattern node that should be analyzed.
      */
     convertBindingPattern(context: Context, node: ts.BindingPattern) {
-        (node.elements as ts.NodeArray<ts.BindingElement>).forEach((element: ts.BindingElement) => {
+        node.elements.forEach((element) => {
             this.owner.convertNode(context, element);
 
-            if (_ts.isBindingPattern(element.name)) {
+            if (!ts.isBindingElement(element)) {
```

Comment:
This additional check is required since the internal `isBindingPattern` check also ensures that `node` exists, while the `is*BindingPattern` do not

Reply:
Would it be better to inline this with the if block below?